public/js/utils.js

```diff
@@ -84,7 +84,7 @@ function submitTransaction () {
     if (err) {
       alert(err)
     } else {
-      window.location.href = '/tx/pending'
+      window.location.href = '/explorer/tx/pending'
```

Comment:
the `/explorer/` prefix should not be hardcoded since it is configurable here: https://github.com/ewasm/etherchain-light/blob/ewasm/config.js#L12 , I think this could be changed to `tx/pending` (without the initial `/`).

Reply:
This is on the frontend.  AFAIK the nodejs config you referenced won't apply here.

Reply:
nodejs config doesn't directly apply, but I think if we set `tx/pending` it would be taken as a relative path and it will redirect accordingly, if we are in path `/deploy` it would redirect to `/tx/pending` if we are in path `/explorer/deploy` it would redirect to `/explorer/tx/pending`.

Reply:
`curl http://40.114.207.112/tx/pending` returns a 404